test/smoke/stacked-block-size-config/stacked_block_size_config.c

```diff
@@ -32,8 +32,8 @@ int main() {
   free(x);
   free(y);
 
-  /// CHECK: DEVID:{{.*}}SGN:5 ConstWGSize:128  args: 2 teamsXthrds:( 938X  16) {{.*}}tripcount:15000 rpc:0
-  /// CHECK: DEVID:{{.*}}SGN:5 ConstWGSize:128  args: 2 teamsXthrds:( 832X 128) {{.*}}tripcount:1000000 rpc:0
+  /// CHECK: DEVID:{{.*}}SGN:5 ConstWGSize:128  args: 2 teamsXthrds:( {{.*}}X  16) {{.*}}tripcount:15000 rpc:0
```

Comment:
@estewart08 @dpalermo
do we have a mechanism like RUN and check-prefixes here to specify multiple (GPU, number of teams) combinations?

@doru1004
Is the team size and number of teams relevant with what this test is trying to check? If not let's remove the whole match of `teamXthrds`.

Reply:
The number of teams is not. This test only cares about the team size.

Reply:
Thanks for the clarification!